app/services/api/teachers/school_transfers/query.rb

```diff
@@ -0,0 +1,83 @@
+module API::Teachers::SchoolTransfers
+  class Query
+    include Queries::FilterIgnorable
+
+    attr_reader :scope
+
+    def initialize(
+      lead_provider_id:,
+      updated_since: :ignore,
+      sort: { created_at: :asc }
+    )
+      @scope = Teacher.distinct
+
+      where_lead_provider_is(lead_provider_id)
+      where_updated_since(updated_since)
+      set_sort_by(sort)
+    end
+
+    def school_transfers
+      preload_associations(block_given? ? yield(scope) : scope)
+    end
+
```

Comment:
@cpjmcquillan Just noticed.. it's missing a method for looking up for single transfer?

Reply:
Not sure we need it for this query as we don't return one transfer anywhere in the API

Reply:
yeah you're right @ethax-ross .. I meant transfers for a single participant `/api/v3/participants/ecf/{id}/transfers`

Reply:
Ah sorry I see what you mean, yes we will need it for that you're right; so this would be finding a single participant with a transfer 👍